atlas_scripts/squirrel_monkey.py

```diff
@@ -0,0 +1,572 @@
+                        name_col = None
+
+                        for col in df.columns:
+                            col_lower = str(col).lower()
+                            if "id" in col_lower and id_col is None:
+                                id_col = col
+                            elif (
+                                any(
+                                    x in col_lower
+                                    for x in ["name", "label", "structure"]
+                                )
+                                and name_col is None
+                            ):
+                                name_col = col
+
+                        if id_col and name_col:
+                            for idx, row in df.iterrows():
+                                try:
+                                    struct_id = int(row[id_col])
+                                    if struct_id == 0 or struct_id == ROOT_ID:
+                                        continue
+
+                                    structures.append(
+                                        {
+                                            "id": struct_id,
+                                            "name": str(row[name_col]),
+                                            "acronym": str(
+                                                row.get(
+                                                    "acronym", row[name_col]
+                                                )
+                                            )[:10],
+                                            "structure_id_path": [
+                                                ROOT_ID,
+                                                struct_id,
+                                            ],
+                                            "rgb_triplet": [
+                                                int(row.get("r", 100)),
+                                                int(row.get("g", 100)),
+                                                int(row.get("b", 100)),
+                                            ],
+                                        }
+                                    )
+                                except (ValueError, KeyError):
+                                    continue
+
+                            if len(structures) > 2:
+                                # More than just root + brain
+                                num_loaded = len(structures) - 2
+                                print(f"Loaded {num_loaded} structures")
+                                break
+                except Exception:
+                    continue
+        except Exception as e:
+            print(f"Error parsing label file: {e}")
+            print("Using default structure list.")
+    else:
+        print("No label definition files found. Using minimal structure list.")
+
+    return structures
+
+
+def create_meshes(download_dir_path, tree, annotated_volume, labels, root_id):
```

Comment:
we introduced a helper function to simplify this. see [the swc_female_rat](https://github.com/brainglobe/brainglobe-atlasapi/blob/main/atlas_scripts/swc_female_rat.py#L149-L163) for an example